blues/example.py

```diff
@@ -35,17 +34,17 @@ def runNCMC(platform_name):
 
     #Define the 'model' object we are perturbing here.
     # Calculate particle masses of object to be moved
-    ligand = ncmc.Model(struct, 'LIG')
+    ligand = RandomLigandRotationMove(struct, 'LIG')
     ligand.calculateProperties()
 
     # Initialize object that proposes moves.
-    ligand_mover = ncmc.MoveProposal(ligand, 'random_rotation', opt['nstepsNC'])
+    ligand_mover = MoveEngine(ligand)
```

Comment:
@nathanmlim - why would you attach the ligand identity and set of moves to the same object (it looks like, in your code, a dictionary...?)?